### PR TITLE
Fix wrong code highlighting for kubectl install on Linux

### PR DESCRIPTION
**Pull request by basefas**
change kubectl install docs code block to console type

### CODE REVIEW

**Comment by tengqm** [comment]
@basefas Please don't submit PRs that span more than one language. We typically merge changes to the English version as the upstream and let the localization teams to decide when and how to synchronize these changes.

**Comment by divya-mohan0209** [comment]
Hello @basefas , thank you for your contributions! We have different processes followed within different localisations of the project. Therefore, I'd request you to make changes to ONLY the main website i.e. the English language with this PR and then raise a separate PR for each localisation you've mentioned here.

**Comment by basefas** [comment]
@divya-mohan0209 @tengqm 
Hi,
Thanks for pointing out the problem. I've changed it in only English version.

**Review by divya-mohan0209 — COMMENTED** [review]
/lgtm

File: content/en/docs/tasks/tools/install-kubectl-linux.md
Comment (dipesh-rawat) [inline content/en/docs/tasks/tools/install-kubectl-linux.md:1]: There is one more place similar change could be made in this file at [Line-Number:261](https://github.com/kubernetes/website/blob/main/content/en/docs/tasks/tools/install-kubectl-linux.md?plain=1#L261)

File: content/en/docs/tasks/tools/install-kubectl-macos.md
Comment (dipesh-rawat) [inline content/en/docs/tasks/tools/install-kubectl-macos.md:1]: There is one more place similar change could be made in this file at [Line-Number:250](https://github.com/kubernetes/website/blob/main/content/en/docs/tasks/tools/install-kubectl-macos.md?plain=1#L250)

File: content/en/docs/tasks/tools/install-kubectl-linux.md
Comment (basefas) [reply content/en/docs/tasks/tools/install-kubectl-linux.md:1]: updated

File: content/en/docs/tasks/tools/install-kubectl-macos.md
Comment (basefas) [reply content/en/docs/tasks/tools/install-kubectl-macos.md:1]: updated

**Comment by basefas** [comment]
Does anything else I should do for this？

**Comment by sftim** [comment]
/retitle Fix wrong code highlighting for kubectl install on Linux

**Review by sftim — COMMENTED** [review]
/lgtm
/approve

**Comment by natalisucks** [comment]
@basefas Because some checks are failing here, if you push an empty commit, it we retrigger the checks and we can re-apply the approvals needed to merge this change. Thanks for sticking with this! 👏

**Comment by sftim** [comment]
I'll manually trigger a new build

**Comment by sftim** [comment]
@basefas, could you rebase against _main_? See https://lwkd.info/2023/20230606#developer-news

**Pull request merged**